### PR TITLE
add protocol tags to docs

### PR DESCRIPTION
**Pull request by blackgirlbytes**
The most significant changes include the addition of a new filter property for record queries, modification of the documentation to explain the new tagging feature, and the inclusion of a new test case to verify the tagging functionality.

Enhancements to record query:

* [`site/docs/web5/build/decentralized-web-nodes/query-from-dwn.mdx`](diffhunk://#diff-d1a221f6540e62798174c2c52168bd3af7409436e609d554053d0c4c5584fd86R52): A new `tags` property was added to the list of filter properties for record queries. This allows for records to be categorized using key-value pairs.

Documentation updates:

* [`site/docs/web5/build/decentralized-web-nodes/write-to-dwn.mdx`](diffhunk://#diff-5752dfa162025e7e670ffd654532efc6e61f1632423d0a9bfd38a693fc7fc6dbL31-R39): The section "Create records from an existing record" was demoted to a sub-section, and a new sub-section titled "Create records with tags" was added. This new section explains how to assign tags to records, enabling filtering by specific keywords.

New test case:

* [`site/testsuites/testsuite-javascript/__tests__/web5/build/decentralized-web-nodes/write-to-dwn.test.js`](diffhunk://#diff-aa48069f8d206290c34cfea31e8b0996e62e800813045d9b0c5d07dc53898338R166-R186): A new test case was added to verify the creation of records with tags. The test ensures that the `tags` property exists in the record and that it contains the correct key-value pairs.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207758653193383

### CODE REVIEW

File: site/docs/web5/build/decentralized-web-nodes/write-to-dwn.mdx [hunk 1]
```diff
@@ -28,9 +28,13 @@ You can also store mixed data via JSON - including text, images, or files.
...
-## Create records from an existing record
+### Create records from an existing record
 You can create a new version of a record while maintaining a link to the version of the original record.
 
 <CodeSnippet snippetName='createRecordFrom'/>
 
+### Create records with tags
+You can assign tags to records to enable filtering by specific keywords.
+<CodeSnippet snippetName='createRecordWithTags'/>
+
```
Comment (blackgirlbytes) [inline site/docs/web5/build/decentralized-web-nodes/write-to-dwn.mdx:39]: is this the appropriate spot for this code snippet?

Comment (acekyd) [reply site/docs/web5/build/decentralized-web-nodes/write-to-dwn.mdx:39]: I believe so.

**Review by acekyd — APPROVED** [review]

File: site/testsuites/testsuite-javascript/__tests__/web5/build/decentralized-web-nodes/write-to-dwn.test.js [hunk 2]
```diff
@@ -162,4 +162,26 @@ test('createRecordFrom creates a record from an existing record', async () => {
...
+test('createRecordWithTags creates a record with tags', async() => {
+   
+  // :snippet-start: createRecordWithTags
+  // Creates a record with tags
+  const { record } = await web5.dwn.records.create({
+    data: "iPhone 13",
+    message: {
+      dataFormat: 'application/json',
+      tags    : {
+        manufacturer: 'Apple',
```
Comment (angiejones) [inline site/testsuites/testsuite-javascript/__tests__/web5/build/decentralized-web-nodes/write-to-dwn.test.js:176]: can you use a generic example not associated with a company?

Comment (blackgirlbytes) [reply site/testsuites/testsuite-javascript/__tests__/web5/build/decentralized-web-nodes/write-to-dwn.test.js]: good point! Fixed!

**Review by angiejones — APPROVED** [review]

**Pull request merged**